Tools/CMake/AMReXInstallHelpers.cmake

```diff
@@ -39,51 +39,53 @@ function (install_amrex_targets)
        ${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIR}/AMReXConfigVersion.cmake
        COMPATIBILITY AnyNewerVersion )
 
-   install( FILES
-      ${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIR}/AMReXConfig.cmake
-      ${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIR}/AMReXConfigVersion.cmake
-      DESTINATION ${CMAKE_FILES_DIR} )
-
-   #
-   # Export install-tree
-   #
-   install(
-      TARGETS       ${_targets}
-      EXPORT        AMReXTargets
-      ARCHIVE       DESTINATION lib
-      LIBRARY       DESTINATION lib
-      INCLUDES      DESTINATION include # Adds proper directory to INTERFACE_INCLUDE_DIRECTORIES
-      PUBLIC_HEADER DESTINATION include
-      RUNTIME       DESTINATION bin
-      )
-
-   install( EXPORT AMReXTargets
-      NAMESPACE AMReX::
-      DESTINATION lib/cmake/AMReX )
-
-   # Install fortran modules if Fortran is enabled
-   get_property(_lang GLOBAL PROPERTY ENABLED_LANGUAGES)
-   if ("Fortran" IN_LIST _lang AND "amrex" IN_LIST _targets)
-      get_target_property(_mod_dir amrex Fortran_MODULE_DIRECTORY )
-      install( DIRECTORY ${_mod_dir}/ DESTINATION include ) # Trailing backslash is crucial here!
-   endif ()
-
-   # Install Tools directory
-   install(
-      DIRECTORY
-      ${PROJECT_SOURCE_DIR}/Tools/CMake
-      ${PROJECT_SOURCE_DIR}/Tools/C_scripts
-      ${PROJECT_SOURCE_DIR}/Tools/typechecker
-      DESTINATION
-      Tools
-      USE_SOURCE_PERMISSIONS
-      )
+   if(AMReX_INSTALL)
+       install( FILES
+          ${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIR}/AMReXConfig.cmake
+          ${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIR}/AMReXConfigVersion.cmake
+          DESTINATION ${CMAKE_FILES_DIR} )
+
+       #
+       # Export install-tree
+       #
+       install(
+          TARGETS       ${_targets}
+          EXPORT        AMReXTargets
+          ARCHIVE       DESTINATION lib
+          LIBRARY       DESTINATION lib
+          INCLUDES      DESTINATION include # Adds proper directory to INTERFACE_INCLUDE_DIRECTORIES
+          PUBLIC_HEADER DESTINATION include
+          RUNTIME       DESTINATION bin
+          )
+
+       install( EXPORT AMReXTargets
+          NAMESPACE AMReX::
+          DESTINATION lib/cmake/AMReX )
+
+       # Install fortran modules if Fortran is enabled
+       get_property(_lang GLOBAL PROPERTY ENABLED_LANGUAGES)
+       if ("Fortran" IN_LIST _lang AND "amrex" IN_LIST _targets)
+          get_target_property(_mod_dir amrex Fortran_MODULE_DIRECTORY )
+          install( DIRECTORY ${_mod_dir}/ DESTINATION include ) # Trailing backslash is crucial here!
+       endif ()
+
+       # Install Tools directory
+       install(
+          DIRECTORY
+          ${PROJECT_SOURCE_DIR}/Tools/CMake
+          ${PROJECT_SOURCE_DIR}/Tools/C_scripts
+          ${PROJECT_SOURCE_DIR}/Tools/typechecker
+          DESTINATION
+          Tools
+          USE_SOURCE_PERMISSIONS
+          )
+   endif()
 
    #
    # Export build-tree
    #
-   export( EXPORT AMReXTargets NAMESPACE AMReX::
```

Comment:
This line only works if an equivalent `install()` line exists for the same target.